PyPDF2/generic/_data_structures.py

```diff
@@ -1041,7 +1247,7 @@ def __init__(
         *args: Any,  # ZoomArgType
     ) -> None:
         DictionaryObject.__init__(self)
-        self[NameObject("/Title")] = title
+        self[NameObject("/Title")] = TextStringObject(title)
```

Comment:
This is a change that is not related to cloning, right? It was just wrong in the tests before.

As this PR is rather big and might take a while to review, I would like to move this to a separate PR ("Miscellaneous fixes" or similar)

Reply:
included in #1411